doc/adrs/XXX_support_representation_details_view.adoc

```diff
@@ -0,0 +1,50 @@
+= ADR-XXX - Add Support for Representation Details View
+
+
+== Context
+
+Currently, when a user selects a representation, nothing is displayed in the details section.
+Representations only have two metadata fields (name and kind).
+Users want more information to distinguish their different representations.
+
+== Decision
+
+To address this need, we will enrich the metadata of representations by adding additional information such as a description and an icon.
+These details will be displayed in the details section when a representation is selected.
+
+=== View DSL
+
+* Enhance the `representationDescription` type to add a description to the representation and the ability to define icons
+
+class RepresentationDescription {
+    EString description
+    InterpretedExpression iconURLsExpression
+}
+
+=== Backend
+
+* Declare a new `IPropertiesDescriptionRegistryConfigurer` whose role will be to display representation metadata in the details view
+
+[source,java]
+----
+public class RepresentationPropertiesDescriptionRegistryConfigurer implements IPropertiesDescriptionRegistryConfigurer {}
+----
+
+This page must only handle representation
+[source,java]
+----
+PageDescription.newPageDescription(UUID.nameUUIDFromBytes("representation_properties_description".getBytes()).toString())
+            //...
+            .canCreatePredicate(this::handles)
+            .build();
+
+ private boolean handles(VariableManager variableManager) {
+        Optional<Object> selectedElement = variableManager.get(VariableManager.SELF, Object.class)
```

Comment:
It may be a RepresentationMetadata instead of an IRepresentation, you'll have to check

Reply:
It seems it's always an IRepresentation to pass as `self `, but it will probably have to be changed to retrieve the representation label, for example.

Reply:
This value will almost certainly change (due to the PR #4142)